src/Components/CreateImageWizardV2/CreateImageWizard.tsx

```diff
@@ -159,6 +163,15 @@ const CreateImageWizard = () => {
           >
             <RegistrationStep />
           </WizardStep>
+          <WizardStep
+            name="OpenSCAP"
+            id="step-oscap"
+            footer={
+              <CustomWizardFooter disableNext={oscapProfile === undefined} />
```

Comment:
I see, so now the user has to choose "None" if they don't want any profile and as "None" is not undefined we can block on undefined.

It seems in the old Wizard we have "None" as the default option, but we also don't block on undefined (which I'm not sure if we should, would have to look deeper into the code).

Would it be possible to set "None" as a default option here as well?

Reply:
remove that, thank you